utils.py

```diff
@@ -41,6 +42,8 @@ def post_status(self, status_json):
 
     @property
     def statuses_url(self):
+        if self.github_proxy is not None:
```

Comment:
```suggestion
        if self.github_proxy:
```

Reply:
https://docs.python.org/2.4/lib/truth.html

Reply:
Was going by https://amir.rachum.com/blog/2012/08/25/you-cant-handle-the-truth/ and consistency within the repo i.e. utils.py on L19

```
        if event is not None:
            self.issue_url = event['pull_request']['issue_url']
```

Reply:
Is it better to be consistent within the application or introduce something new?

Reply:
Its better to be consistent